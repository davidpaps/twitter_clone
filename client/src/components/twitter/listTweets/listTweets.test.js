import React from "react";
import { shallow } from "enzyme";
import ListTweets from "./listTweets";
import renderer from "react-test-renderer";

describe("List Tweet", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<ListTweets />)));

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(1);
  });

  it("should render correctly", () => {
    const tree = renderer.create(<ListTweets />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
