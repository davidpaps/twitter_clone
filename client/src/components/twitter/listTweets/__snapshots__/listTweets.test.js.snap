// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Tweet should render correctly 1`] = `
<h1
  className="text-center mt-5"
>
   Your Twitter Feed:
</h1>
`;
